lib/stm32/h7/Makefile

```diff
@@ -38,12 +38,12 @@ TGT_CFLAGS	+= $(STANDARD_FLAGS)
 ARFLAGS		= rcs
 
 OBJS += dac_common_all.o
+OBJS += exti_common_all.o
 OBJS += flash_common_all.o flash_common_f.o flash_common_f24.o
 OBJS += fmc_common_f47.o
 OBJS += gpio_common_all.o gpio_common_f0234.o
 OBJS += pwr.o rcc.o
 OBJS += rcc_common_all.o
-OBJS += spi_common_all.o spi_common_v2.o
```

Comment:
this seems out of scope?

Reply:
I always have some fun issues cherry-picking across our staging branch to single-feature branches...this should now be untouched.